addons/hr/models/hr_department.py

```diff
@@ -132,6 +141,28 @@ def action_plan_from_department(self):
         action['context'] = {'default_department_id': self.id, 'search_default_department_id': self.id}
         return action
 
+    def action_employee_from_department(self):
```

Comment:
check get_view on hr_employee

Reply:
I had some issue because search_view are differents for the two models.

Reply:
As we discussed i didn't find an other good solution